app/controllers/questions_controller.rb

```diff
@@ -4,13 +4,25 @@ def index
   end
 
   def create
+    logger.debug "PARAMS for create: #{params}"
     question = Question.new(params.require(:question).permit(:text, :user_id))
+    logger.debug "QUESTION created: #{question}"
+    logger.debug "ID of Question created: #{question.id}"
```

Comment:
Is there an ID before you do `question.save`? I think you'll have to move this line below the `.save` method (you can test this by looking at the output -- no ID, right?)